client/src/components/admin/AdminRound/AdminRoundContent.js

```diff
@@ -135,6 +135,7 @@ function AdminRoundContent({ round, competitionId }) {
             focusOnResultChange={true}
             disabled={loading}
             onSubmit={handleResultAttemptsSubmit}
+            officialWorldRecords={officialWorldRecords}
           />
```

Comment:
We also use the form in `RoundDoubleCheck`, so please update it accordingly :)

Reply:
Oops! I checked if `AdminRoundContent` had been used anywhere else but not the form. Fixed!